scripts/clone_vendors.py

```diff
@@ -79,9 +82,18 @@ async def handle_generate(language_name: str, directory: str | None, abi_version
         if directory
         else (vendor_directory / language_name).resolve()
     )
+    
+    # 윈도우 환경에서는 cmd.exe /c를 통해 명령어를 실행해야 PATH에서 tree-sitter를 찾을 수 있음
```

Comment:
please use english or remove the comment - also it should only be essential, if the comment can be understood from the code, no comment is required.